apps/masterbots.ai/lib/hooks/useSonner.tsx

```
@@ -0,0 +1,84 @@
+
+export type customSonnerProps = {
+  type?: 'success' | 'error' | 'info'
+  text?: string | null
+}
+
+export function useSonner() {
+  let config: {
+    title: string
+    text: string
+    button: React.ReactNode | string
+    icon: React.ReactNode | any
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)